src/components/menuPage/ListFormat.js

```diff
@@ -0,0 +1,26 @@
+const ListFormat = (props) => {
+    return ( 
+        props.tagArr
+        .map((tag)=>{
+            return(
+                <div id={tag} key={tag} className="mx-2 mt-4 mb-5">
+                <h3 className="m-2">{tag.toUpperCase()}</h3>
+                <div className="scrollMenu">
```

Comment:
When I reduce the width of the browser, I can not scroll to the right to see other menu items.

Reply:
Video: https://drive.google.com/file/d/135frRcBa8WP1a2dGpfpV_xa5p3aYHQu4/view?usp=sharing

Reply:
How does the scrolling work? Does it use a library?

Reply:
Only CSS. Please check app.css